graph/telemetry/istio/appender/ambient.go

```diff
@@ -34,11 +34,13 @@ func (a AmbientAppender) AppendGraph(trafficMap graph.TrafficMap, globalInfo *gr
 	log.Trace("Running ambient appender")
 
 	if !a.Waypoints {
-		a.removeWaypointEntries(trafficMap, globalInfo, namespaceInfo)
+		a.isWaypoint(trafficMap, globalInfo, true)
+	} else {
+		a.isWaypoint(trafficMap, globalInfo, false)
 	}
```

Comment:
Fine as is, but could reduce to just one line:

`a.isWaypoint(trafficMap, globalInfo, !a.Waypoints)`

Reply:
oh! Good catch